src/containers/SortingView.tsx

```diff
@@ -66,7 +66,7 @@ const SortingView: React.FunctionComponent<Props> = (props) => {
   //     const sortingInfo = await createHitherJob(
   //       'createjob_get_sorting_info',
   //       { sorting_object: sorting.sortingObject, recording_object: recording.recordingObject },
-  //       { kachery_config: {}, useClientCache: true, wait: true, newHitherJobMethod: true}
+  //       { kachery_config: {}, useClientCache: true }
```

Comment:
Nice not to have to specify the `newHitherJobMethod` everywhere too.

Reply:
yes... everything is now ported to new hither job method.